blockpoller/state_file_test.go

```diff
@@ -33,7 +33,7 @@ func TestFireBlockFinalizer_state(t *testing.T) {
 	fk.AddLink(bstream.NewBlockRef("106a", 106), "105a", &block{Block: blk("106a", "105a", 101)})
 	fk.AddLink(bstream.NewBlockRef("105a", 105), "104a", &block{Block: blk("105a", "104a", 101)})
 	expectedBlocks, reachedLib := fk.CompleteSegment(blk("105a", "104a", 101).AsRef())
-	// simulate firing the blocks
+	// simulate firing the optimisticlyPolledBlocks
```

Comment:
You search/replace was too broad, please review all renames and adjust them when it's not needed.